src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -401,12 +408,15 @@ struct SvgLoaderData
     Array<SvgNode *> stack = {nullptr, 0, 0};
     SvgNode* doc = nullptr;
     SvgNode* def = nullptr;
+    SvgNode* cssStyle = nullptr;
     Array<SvgStyleGradient*> gradients;
     SvgStyleGradient* latestGradient = nullptr; //For stops
     SvgParser* svgParse = nullptr;
     Array<SvgNodeIdPair> cloneNodes;
+    Array<SvgNodeIdPair> cloneCssStyleNodes;
     int level = 0;
     bool result = false;
+    bool style = false; //TODO: find a better sollution?
```

Comment:
when parsing data or cdata, I need to know what was the previous tag. if it was 'style' I can assume that the data describe the style.  otherwise tvg ignores it.

Reply:
please check the example:
```
<svg viewBox="0 0 200 200">
    <defs>
        <style>
        <![CDATA[
            rect.mineStyle {fill: blue}
        ]]>
        </style> 
    </defs>

    <![CDATA[ 
        rect {stroke-width: 15; stroke: red}
    ]]>

    <rect x="0" y="0" width="50" height="50" class="mineStyle" />
</svg>
```

The second CDATA block should be skipped. 
While it is read, `loader->cssStyle != nullptr` and `loader->style = false`., so checking the `loader->cssStyle` value won't be enough.
Although I pushed a commit, since the `loader->style` value was reset in a wrong place. fixed now